### PR TITLE
Update project name to Argo Events

### PR DESCRIPTION
**Pull request by edlee2121**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
-# Axis - The Event-Based Dependency Manager for Kubernetes
+# Argo Events - The Event-Based Dependency Manager for Kubernetes
 [![Build Status](https://travis-ci.org/blackrock/axis.svg?branch=master)](https://travis-ci.org/blackrock/axis) [![License Apache 2.0](https://img.shields.io/badge/License-Apache2-brightgreen.svg)](https://img.shields.io/badge/License-Apache2-brightgreen.svg)
```
Comment (magaldima) [inline README.md]: We probably want to change these badges as well. Also we want to remove the `.travis-ci.yml` and put in a `.argo-ci/ci.yaml` file instead.

File: docs/index.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
-# Axis Documentation
+# Argo Events Documentation
 
-## Why Axis?
-- Containers. Axis is designed from the ground-up as Kubernetes native.
+## Why Argo Events??
```
Comment (magaldima) [inline docs/index.md]: remove one `?`

File: docs/index.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+- Containers. Designed from the ground-up as Kubernetes0native.
```
Comment (magaldima) [inline docs/index.md]: spell-check `Kubernetes0native`

File: docs/quickstart.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ This is a guide to getting started with using Axis using Minikube.
 
 ## 1. Checkout project's master branch
 ```
-$ git clone git@github.com:/axis.git
+$ git clone git@github.com:argoproj/argo-events.git blackrock/axis
```
Comment (magaldima) [inline docs/quickstart.md]: do we need `blackrock/axis` appended after clone?

File: docs/quickstart.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ $ minikube start
 $ eval $(minikube docker-env)
 ```
 
-## 5. Build the Axis project & Docker images
+## 5. Build the project & Docker images
 ```
 $ cd go/src/github.com/blackrock/axis
```
Comment (magaldima) [inline docs/quickstart.md]: probably want to either update this or remove it altogether

**Comment by magaldima** [comment]
I've updated the project refs and remove the travis build badge in #30 . Can you rebase on top of that?

File: README.md (hunk 1 above)
Comment (edlee2121) [reply README.md]: We should discuss with Jesse and Alexander.

File: docs/index.md (hunk 2 above)
Comment (edlee2121) [reply docs/index.md]: Done.

File: docs/index.md (hunk 3 above)
Comment (edlee2121) [reply docs/index.md]: Done

File: docs/quickstart.md (hunk 4 above)
Comment (edlee2121) [reply docs/quickstart.md]: Done.

**Comment by edlee2121** [comment]
Updated PR

File: README.md (hunk 1 above)
Comment (magaldima) [reply README.md]: okay, we'll tackle this outside this PR

**Pull request merged**